src/org/zaproxy/zap/extension/stats/OptionsStatsPanel.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2016 ZAP development team
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.stats;
+
+import java.awt.CardLayout;
+import java.awt.GridBagLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.net.InetAddress;
+
+import javax.swing.BorderFactory;
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.TitledBorder;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.model.OptionsParam;
+import org.parosproxy.paros.view.AbstractParamPanel;
+import org.zaproxy.zap.utils.FontUtils;
+import org.zaproxy.zap.utils.ZapPortNumberSpinner;
+import org.zaproxy.zap.utils.ZapTextField;
+import org.zaproxy.zap.view.LayoutHelper;
+
+public class OptionsStatsPanel extends AbstractParamPanel {
+
+	private static final long serialVersionUID = 1L;
+	private JPanel panelMisc;
+	private JCheckBox inMemoryStatsEnabledField;
+	private JCheckBox statsdStatsEnabledField;
+	private ZapTextField statsdHostField; 
+	private ZapPortNumberSpinner statsdPortField;
+	private ZapTextField statsdPrefixField;
+
+    public OptionsStatsPanel() {
+        super();
+        this.setLayout(new CardLayout());
+        this.setName(Constant.messages.getString("stats.options.title"));
+        this.add(getPanelMisc(), getPanelMisc().getName());
+	}
+
+	/**
+	 * This method initializes panelMisc	
+	 * 	
+	 * @return javax.swing.JPanel	
+	 */    
+	private JPanel getPanelMisc() {
+		if (panelMisc == null) {
+			panelMisc = new JPanel();
+			panelMisc.setLayout(new GridBagLayout());
+			panelMisc.add(getInMemoryStatsEnabledField(), LayoutHelper.getGBC(0, 0, 1, 0.5));
+
+			panelMisc.add(getStatsdStatsEnabledField(), LayoutHelper.getGBC(0, 1, 1, 0.5));
+						
+			JPanel statsdPanel = new JPanel();
+			statsdPanel.setLayout(new GridBagLayout());
+			statsdPanel.setBorder(
+					BorderFactory.createTitledBorder(
+							null, Constant.messages.getString("stats.options.statsd.panel"), 
+							TitledBorder.DEFAULT_JUSTIFICATION,
+							javax.swing.border.TitledBorder.DEFAULT_POSITION,

```

Comment:
`javax.swing.border.TitledBorder` is imported, couldn't this be `TitledBorder.DEFAULT_POSITION` for brevity/consistency (with L77).